src/k8s-engine/minikube.js

```diff
@@ -60,12 +60,22 @@ class Minikube {
       opts.env['MINIKUBE_HOME'] = paths.data();
 
       // TODO: Handle platform differences
-      let args = ['start', '-p', 'rancher-desktop', '--driver', 'hyperkit', '--container-runtime', 'containerd', '--interactive=false'];
+      let args = ['start', '-p', 'rancher-desktop', '--container-runtime', 'containerd', '--interactive=false'];
+      switch (os.platform()){
+        case "linux":
+          args.push('--driver', 'docker')
```

Comment:
Unfortunately, we can't assume Docker is installed. We want this for the case where there isn't even docker installed. Our goal is to use k3s in all environments with containerd.

I realize this complicates the Linux usage. I'm open to chatting about this on a call.

Reply:
Isnt this like the current mac implementation with hyperkit? You expect the host to have some requirements, otherwise it wont work?

In any case, using k3s where available makes sense, but that would require a big expansion to the current code, which is out of scope for my goal (understanding how rd works underneath)